tests/foreman/ui_airgun/test_os.py

```diff
@@ -34,8 +36,15 @@ def test_positive_create_with_arch(session):
 def test_positive_create_with_ptable(session):
     name = gen_string('alpha')
     major_version = gen_string('numeric', 2)
-    ptable = entities.PartitionTable().create()
+    org = entities.Organization().create()
+    loc = entities.Location().create()
+    ptable = entities.PartitionTable(
+        organization=[org],
+        location=[loc],
+    ).create()
     with session:
+        session.organization.select(org_name=org.name)
```

Comment:
hm. wanted to propose different fix - with session_org and default org for user. But for demo purposes your approach may appear even better.

Reply:
right, and your idea is completely out of scope for the task achived

Reply:
exactly, that's why it's not review comment but just a separate one 🙂